src/main/kotlin/racingcar/utils/Round.kt

```diff
@@ -0,0 +1,17 @@
+package racingcar.utils
+
+import camp.nextstep.edu.missionutils.Randoms
+
+object Round {
+    internal const val RANDOM_RANGE_START = 0
+    internal const val RANDOM_RANGE_END = 9
+
+    internal const val FORWARD_THRESHOLD = 4
+
+    internal fun generateRandomNumber() =
+        Randoms.pickNumberInRange(RANDOM_RANGE_START, RANDOM_RANGE_END)
+
+    internal fun isForward(randomNumber: Int): Boolean =
+        true.takeIf { randomNumber >= FORWARD_THRESHOLD } ?: false
+
```

Comment:
간단하게 아래와 같이 바꿔보면 어떨까요?

```suggestion
    internal fun isForward(randomNumber: Int): Boolean = randomNumber >= FORWARD_THRESHOLD

```

Reply:
GPT에게 코드를 더 효율적으로, 짧게 줄일 수 있는 방법을 물어보면 takeIf를 자주 추천해주는 것 같아요. takeIf 함수가 사용하면 안 되는 나쁜 함수는 아니지만, 자주 사용된다면 GPT 스럽다는 느낌을 줄 것 같아요.
 취업 관련 수업을 듣고 있는데, 최근에는 서류를 AI로 돌려서 1차로 거르는 시스템도 만들어졌다고 하네요. 코드는 아직 사람이 읽어보고, AI가 takeIf를 좋게 볼 수도 있겠지만요!

 또한, 이전에 takeIf 관련해서 아래와 같이 코드 리뷰를 받은 적이 있습니다!
 https://github.com/DoTheBestMayB/UpbitAPI/pull/35#discussion_r882635648

> A.
> takeIf는 가독성이 떨어져서 사용하는 것을 지양하는 편입니다.
> takeIf를 사용하지 않고 단순 if문을 사용해서 코드를 작성해주세요.
> 
> 그리고 범위 연산자들을 다양하게 사용해주셨는데, 범위 연산자 남용은 오히려 코드 가독성이 떨어져요.
> 최대한 간결하게 작성해보세요 💪
> 
> Q. takeIf는 작성한 사람, 읽는 사람 모두 단순히 코드만 보았을 때 어떤 의미인지 헷갈릴 수 있어서 그런가요?
>
> A.
> 네 맞습니다.
> 이 부분은 이펙티브 코틀린에서도 안내가 되어있어요.
> 아이템 11. 가독성을 목표로 설계하라 <- 이 부분을 읽어보세요.